backend/dataall/modules/worksheets/aws/bedrock_client.py

```diff
@@ -0,0 +1,51 @@
+from dataall.base.aws.sts import SessionHelper
+from langchain_aws import ChatBedrock as BedrockChat
+from langchain_core.output_parsers import StrOutputParser
+from langchain_core.prompts import PromptTemplate
+from dataall.base.db import exceptions
+import os
+
+TEXT_TO_SQL_EXAMPLES_PATH = os.path.join(os.path.dirname(__file__), 'bedrock_prompts', 'text_to_sql_examples.txt')
+TEXT_TO_SQL_TEMPLATE_PATH = os.path.join(os.path.dirname(__file__), 'bedrock_prompts', 'test_to_sql_template.txt')
+PROCESS_TEXT_TEMPLATE_PATH = os.path.join(os.path.dirname(__file__), 'bedrock_prompts', 'process_text_template.txt')
+
+
+class BedrockClient:
+    def __init__(self):
+        self._session = SessionHelper.get_session()
+        self._client = self._session.client('bedrock-runtime')
+        model_id = 'anthropic.claude-3-5-sonnet-20240620-v1:0'
```

Comment:
I am facing errors of the type: ` An error occurred (ValidationException) when calling the InvokeModel operation: Invocation of model ID anthropic.claude-3-5-sonnet-20240620-v1:0 with on-demand throughput isn’t supported. Retry your request with the ID or ARN of an inference profile that contains this model.` I think it is related to the enforced [cross-region inference](https://docs.aws.amazon.com/bedrock/latest/userguide/cross-region-inference.html)

Reply:
I wonder if you have faced something similar when using this model